### PR TITLE
List Ops: drop fold function ordering from the problem spec

### PR DESCRIPTION
**Pull request by IsaacG**
See https://forum.exercism.org/t/misleading-foldr-tests-in-python-track-list-ops/6215/11 for discussion

### CODE REVIEW

File: exercises/list-ops/description.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ The precise number and names of the operations to be implemented will be track d
 - `filter` (*given a predicate and a list, return the list of all items for which `predicate(item)` is True*);
 - `length` (*given a list, return the total number of items within it*);
 - `map` (*given a function and a list, return the list of the results of applying `function(item)` on all items*);
-- `foldl` (*given a function, a list, and initial accumulator, fold (reduce) each item into the accumulator from the left using `function(accumulator, item)`*);
-- `foldr` (*given a function, a list, and an initial accumulator, fold (reduce) each item into the accumulator from the right using `function(item, accumulator)`*);
-- `reverse` (*given a list, return a list with all the original items, but in reversed order*);
+- `foldl` (*given a function, a list, and initial accumulator, fold (reduce) each item into the accumulator from the left using the function*);
```
Comment (vaeng) [inline exercises/list-ops/description.md:15]:
```suggestion
- `foldl` (*given a function, a list, and initial accumulator, fold (reduce) each item into the accumulator from the left*);
```
I feel like dropping the last part feels more natural.

File: exercises/list-ops/description.md [hunk 2]
```diff
@@ -12,6 +12,8 @@ The precise number and names of the operations to be implemented will be track d
...
+- `foldr` (*given a function, a list, and an initial accumulator, fold (reduce) each item into the accumulator from the right using the function*);
```
Comment (vaeng) [inline exercises/list-ops/description.md:16]:
```suggestion
- `foldr` (*given a function, a list, and an initial accumulator, fold (reduce) each item into the accumulator from the right*);
```

**Review by kotp — APPROVED** [review]
Approving with the acknowledgement about dropping the mention of using the function/procedure/method/code that is created.

**Review by ErikSchierboom — APPROVED** [review]
I like it

**Review by vaeng — APPROVED** [review]

**Pull request merged**